src/main/java/de/srendi/advancedperipherals/AdvancedPeripherals.java

```diff
@@ -67,4 +69,11 @@ public void commonSetup(FMLCommonSetupEvent event) {
         });
     }
 
+    public void onLoadComplete(FMLLoadCompleteEvent event) {
+        event.enqueueWork(() -> {
```

Comment:
Do we maybe want to put this into the APAddons class or into the Integration class direclty

Do we even need a `Integration` class if the run method is not used? Maybe we want to re-organize that

Reply:
I was trying to do that in Integration.run(), but Integration runs before items get registered...
I'll move them to Registries

Reply:
That is weird
Should be happen after registration
https://github.com/IntelligenceModding/AdvancedPeripherals/blob/016c6c2e3846b6732e7a5cd0afa7308abdf6cc91/src/main/java/de/srendi/advancedperipherals/common/setup/APRegistration.java#L48